csrc/runtime/fusion_kernel_runtime.cpp

```diff
@@ -330,6 +330,45 @@ KernelArgumentHolder FusionKernelRuntime::runWithInputs(
   return fusion_outputs;
 }
 
+std::vector<KernelArgumentHolder> FusionKernelRuntime::prepareInputs(
```

Comment:
Factored out the input preparation step as suggested by @wujingyue 

https://github.com/NVIDIA/Fuser/pull/4373/files#r2074246473